dowhy/causal_refuters/non_parametric_sensitivity_analyzer.py

```diff
@@ -0,0 +1,172 @@
+import scipy
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.model_selection import KFold
+from econml.utilities import cross_product
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.linear_model import LinearRegression, Lasso
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.compose import ColumnTransformer, make_column_selector
+from dowhy.utils.util import get_numeric_features
+from dowhy.causal_refuters.partial_linear_sensitivity_analyzer import PartialLinearSensitivityAnalyzer
+from dowhy.causal_refuters.reisz import ReiszRegressor, ReiszRepresenter, generate_moment_function, get_alpha_estimator, get_generic_regressor, create_polynomial_function
+
+
+class NonParametricSensitivityAnalyzer(PartialLinearSensitivityAnalyzer):
+    """
+    Class to perform Non parametric Senitivity Analysis

```

Comment:
you need to repeat/copy all the parameters that this class expects from the partiallinear class. This is good for the user to understand. Make sure to remove those that are not required by nonparametric.